apps/src/applab/DesignToggleRow.jsx

```diff
@@ -69,13 +105,22 @@ module.exports = React.createClass({
     }
 
     return (
-      <div>
+      <div style={wrapperStyle} className="justify-contents">
+        <button
+            id='codeModeButton'
+            style={codeButtonStyle}
+            className='no-outline'
+            onClick={this.handleSetMode.bind(this, Mode.CODE)}>
+          {msg.codeMode()}
+        </button>
         <button
-          id="designModeToggle"
-          className="share"
-          onClick={this.handleModeToggle}>
-          { this.state.mode === Mode.DESIGN ? msg.codeMode() : msg.designMode() }
+            id='designModeButton'
+            style={designButtonStyle}
+            className='no-outline'
+            onClick={this.handleSetMode.bind(this, Mode.DESIGN)}>
+          {msg.designMode()}
         </button>
+        {' ' /* Needed for "text-align: justify;" to work. */ }
```

Comment:
that is correct. there are a few styles (e.g. text-align, line-height) that got pulled out into css as well as collateral damage, because I wanted that class to contain all the logic for justifying the contents, so that i could give it a sensible name.